src/resources/views/crud/fields/checklist.blade.php

```diff
@@ -25,7 +25,7 @@
     <label>{!! $field['label'] !!}</label>
     @include('crud::fields.inc.translatable_icon')
 
-    <input type="hidden" value='@json($field['value'])' name="{{ $field['name'] }}">
+    <input type="hidden" value='@if($field['value'])@json($field['value'])@endif' name="{{ $field['name'] }}">
```

Comment:
Wouldn't it be cleaner (and arguably more "correct") to have an empty array instead of an empty string? So something like:

```suggestion
    <input type="hidden" value='@json($field['value'] ?? [])' name="{{ $field['name'] }}">
```

Reply:
@pxpm what do you think?

Reply:
@tabacitu I think it doesn't work.

`$field['value']` is `""`
`$field['value'] ?? []` is `""`

Reply:
`$field['value']` is `""` because on it's definition it fallback to `""`, and that probably is problem.
It there anyway this should be an empty string?
Is it safe to change it to `[]`?

That would be the correct fix.

Reply:
Agree with @promatik . 

But also agree with @tabacitu that it could be "cleaner"

My proposal:

```php
<input type="hidden" value='@json(!empty($field['value']) ? $field['value'] : [])' name="{{ $field['name'] }}">

//it cant work with !empty($field['value']) ?: [] because it would return 1, the result of !empty($field['value'])
```

Reply:
@pxpm I was testing it now, and everything works by changing only the fallback to Array.
I tested changing values with errors, to test the `old()` method, and it works too.

I'll push a commit to this branch, can you take a look?